src/server/migrations/0.6.0-0.7.0/index.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const database = require('../../models/database');
+const sqlFile = database.sqlFile;
+
+module.exports = {
+	fromVersion: '0.6.0',
+	toVersion: '0.7.0',
+	up: async db => {
+		await db.none(sqlFile('../migrations/0.6.0-0.7.0/sql/group/add_displayable_gps_note_area_columns.sql'));
+		await db.none(sqlFile('../migrations/0.6.0-0.7.0/sql/map/add_north_angle_and_circle_fraction.sql'));
+		await db.none(sqlFile('../migrations/0.6.0-0.7.0/sql/meter/add_meter_type.sql'));
+		await db.none(sqlFile('../migrations/0.6.0-0.7.0/sql/meter/add_multiple_db_meters_columns.sql'));
+		await db.none(sqlFile('../migrations/0.6.0-0.7.0/sql/user/add_user_role.sql'));
+	}
```

Comment:
Why does the path leave the current directory and then reenter it? Why not just './sql/....'. Same question for migrations 0.5.0 - 0.6.0

Reply:
I think this is because sqlFile() works relative to the sql/ directory. The code comment states:
// ... the path to the sql file, relative to the sql directory
so this is first going up from the sql/ directory to the server/ directory and then down into the migrations directory. As such, I think it is correct.